spec/services/adapters/structs/income_details_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe Adapters::Structs::IncomeDetails do
   subject { application_struct.income }
 
-  let(:application_struct) { build_struct_application }
+  let(:application_struct) { build_struct_application(with_full_means: false) }
```

Comment:
Do we have a test in here that also checks when we do have full means?

Reply:
Well spotted. I disabled it when debugging. Have sorted now. Thank you :)